src/utils/sqrt.js

```diff
@@ -0,0 +1,23 @@
+const MAX_DECIMAL = 7;
+
+const roundFloat = (num, result, maxDecimal = MAX_DECIMAL) => {
```

Comment:
maybe should this function return a rounded value? Then this function will use only 2 parameters and do only one thing - round value